app/components/bali/notification/component.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+module Bali
+  module Notification
+    class Component < ApplicationViewComponent
+      attr_reader :options
+
+      #
+      # Notification Component with different type of notification.
+      #
+      # @param [<Symbol>] type This adds a class for the notification: :success, :danger
+      # @param [<Integer>] delay How long the notification will be shown.
+      # @param [<Hash>] options This adds a custom attributes to the component.
+
```

Comment:
Ya te habia dejado este comentario

```suggestion
      # Notification Component with different type of notification.
      #
      # @param [Symbol] type This adds a class for the notification: :success, :danger
      # @param [Integer] delay How long the notification will be shown.
      # @param [Hash] options This adds a custom attributes to the component.
```